### PR TITLE
Add an implementation of a custom ROSJava Service

### PR DESCRIPTION
**Pull request by vmoysiadis**

### CODE REVIEW

**Comment by vmoysiadis** [comment]
I signed it!

**Comment by jubeira** [comment]
@vmoisiadis thanks for submitting this! I gave a second thought about this and I think that maybe we shouldn't place this code here. 
Let me explain: this code is already in the tutorial here: http://wiki.ros.org/rosjava_build_tools/Tutorials/indigo/UsingServices. This package will require artifacts for the messages (see this line: https://github.com/rosjava/rosjava_core/pull/260/files#diff-54515f50ed0b12aa7ed4266b813d7d48R39), which are not in rosjava_mvn_repo, and it wouldn't make sense for them to be there. 
I think the idea of the tutorial is to guide users to create new custom messages from scratch -and the tutorial already does that by itself-, but if we place the artifacts in rosjava_mvn_repo, then it's not required to generate them locally. And we already have an example of a Client/ Server for a service in the standard [rosjava_tutorial_services](https://github.com/rosjava/rosjava_core/tree/kinetic/rosjava_tutorial_services) using mvn artifacts.

To sum up, the tutorial should be enough by itself to show the point, and adding the code here doesn't add any additional information for someone that needs to learn how to create a new service from scratch.
Unless you have a different strong opinion against this or unless you think I'm missing something, I'd close this PR.

**Comment by vmoysiadis** [comment]
Hi! I agree with you, maybe is not necessary, you can close the PR
